tests/unit/specs/components/common/TmBalance.spec.js

```diff
@@ -43,7 +43,7 @@ describe(`TmBalance`, () => {
     wrapper.setData({ overview: false })
```

Comment:
this test is wrong. when not signed in, this should show nothing?

Reply:
it will show `Total ATOM` but not `Available ATOM`

Reply:
`show empty stakes when not signed in`
this is wrong. when not signed in, this should not show anything. we never show this component when not signed in any ways.

What you want is to change the test so it does not display `Available ATOM` when there are no tokens. Please pay attention to the tests and not just fix them so they pass.

Reply:
this whole test doesn't make sense then. "show empty stakes when not signed in" means don't show anything. so we can just use the snapshot - yeah?

Reply:
> What you want is to change the test so it does not display Available ATOM when there are no tokens. Please pay attention to the tests and not just fix them so they pass.